.github/workflows/harvest-post-raw-main.yml

```
@@ -0,0 +1,54 @@
+name: Harvest Post Raw Main
```

```suggestion
name: Harvest Post Raw Vanilla
```

Changing the filename is also preferred.

---

fixed: https://github.com/vulsio/vuls-data-db/compare/dcbef5ab05c1e7d6012ff2f63ee2cae65eb39622..13f900f7c04514c90d72788e929a9a69df4dc780